pkg/kube/snapshot/mocks/mock_snapshotter.go

```diff
@@ -1,18 +1,17 @@
 // Code generated by MockGen. DO NOT EDIT.
-// Source: github.com/kanisterio/kanister/pkg/kube/snapshot (interfaces: Snapshotter)
+// Source: snapshot.go
 
-// Package mocks is a generated GoMock package.
+// Package mock_snapshot is a generated GoMock package.
```

Comment:
Please correct the package name, also the the `Source:` changes seems to be incorrect/unnecessary